script1.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
+const nameInput = document.getElementById('nameInput');
+const authorInput = document.getElementById('authorInput');
+const addBtn = document.querySelector('.add-btn');
+const storageKey = 'booksLocalData';
+const loadedBooks = JSON.parse(localStorage.getItem(storageKey)) || [];
+const bookslist = document.querySelector('.books-list');
+
+function loadBook() {
+  bookslist.innerHTML = '';
+  if (loadedBooks != null) {
+    for (let i = 0; i < loadedBooks.length; i += 1) {
+      bookslist.innerHTML += `<div class="book-info">
+              <p>${loadedBooks[i].name}</p>
+              <p>${loadedBooks[i].author}</p>
+              <button class="remove-btn" id="btn-${i}" onclick="deleteBook(${i})" value="remove">Remove</button>
+              <hr>
+              </div>`;
+    }
+  }
+}
+
+window.onload = () => {
+  loadBook();
+};
+
+function addBook() {
+  const book = { name: nameInput.value, author: authorInput.value };
+  loadedBooks.push(book);
+  localStorage.setItem(storageKey, JSON.stringify(loadedBooks));
+  loadBook();
+  nameInput.value = '';
+  authorInput.value = '';
```

Comment:
- [ ] The `addBook` function is not working as expected. The new book is not being added to the `loadedBooks` array. Kindly see the image attached below for more information 👇 
![image](https://user-images.githubusercontent.com/46839548/206189028-928e91e8-8f5e-4fe8-a2cb-a95da41ab74c.png)